spec.md

```diff
@@ -1430,7 +1430,7 @@ Platforms SHOULD initiate orphan mitigation in the following scenarios:
 | 201 | Success | No |
 | 201 with malformed response | Failure | Yes |
 | All other 2xx | Failure | Yes |
-| 408 | Timeout failure | Yes |
+| 408 | Client timeout failure (request not received at the server) | No |
```

Comment:
I don't understand the usage of 'server' here. A client could get a 408 by having it's local timeout setting very small. Or you could be talking to a broker proxy and the second connection could timeout. Both cases could result in orphans. I am not sure how a broker author could prevent their implementation from returning a 408 AND/OR make sure that orphans are not possible.

Reply:
perhaps "receiver" would be better, but in the other cases you mentioned I'm not sure the Platform would see a 408. As long as the message makes it to the next hop I don't think 408 would be appropriate. And I would think that hop before that one would return a 5xx since from its POV it did receive the entire request so it can't be a 408.  Right?

Reply:
Looking up [408 in the rfc](https://tools.ietf.org/html/rfc7231#section-6.5.7)...

Ok I agree. In this case, server is fine with me.